js/index.js

```diff
@@ -60,8 +62,16 @@ import("../crate/pkg")
      */
     function update(d) {
       delta = d;
-      updateBalls(balls["wasm"]);
-      updateBalls(balls["js"]);
+      switch (stage.mode) {
+        case "wasm-compute-js-render-canvas":
+        case "wasm-compute-js-render-html":
+          updateBalls(balls["wasm"]);
+          break;
+        case "js-compute-js-render-canvas":
+        case "js-compute-js-render-html":
+          updateBalls(balls["js"]);
+          break;
+      }
```

Comment:
Keep only this block of changes and remove the others - see the explanations on the conversation tab.